sample-storage/connections.yaml

```diff
@@ -78,3 +78,11 @@
     database: plotly
     port: 50000
     host: 35.184.35.183
+
+-
+    dialect: apache spark
+    id: apache spark-7b7bfa1e-cdfa-4001-b326-e0e170667021
+    timeout: 180
+    database: plotly
+    port: 8998
+    host: 104.155.170.7
```

Comment:
should there be another one in here for the sample IBM DB? 

It's not really documented, but `sample-storage` is used for the test heroku instance (heroku doesn't have a persistent file system, so we need to commit these credentials as part of the repo and then point the settings folder to this `sample-storage` folder to preload the available connections).

Reply:
NVM, it's already there :woman_facepalming: